code/game/objects/items/defib.dm

```diff
@@ -344,12 +346,13 @@
 	if(!req_defib || !defib)
 		return
 	if(!in_range(src,defib))
-		var/mob/living/L = loc
-		if(istype(L))
-			to_chat(L, "<span class='warning'>[defib]'s paddles overextend and come out of your hands!</span>")
+		var/mob/living/user = loc
+		if(istype(user))
+			to_chat(user, "<span class='warning'>[defib]'s paddles overextend and come out of your hands!</span>")
+			dropped(user)
```

Comment:
This is a proc you call after the event has happened, not to cause it to drop.

Reply:
Good point, this is not even used in this fix now.
Cleaning this up now.